index.html

```diff
@@ -33,7 +33,7 @@
     <!-- Navigation -->
     <nav class="navbar fixed-top navbar-expand-lg navbar-dark bg-dark fixed-top">
       <div class="container">
-        <a class="navbar-brand" href="index.html">Start Bootstrap</a>
+        <a class="navbar-brand" href="index.html">Poet's Page</a>
         <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
```

Comment:
I see the change made to Poet's Page on my live server, so I know that's working. Nice work.
It looks like we're going with Modern Bootstrap, but internally it seems like there's still a question about which Bootstrap template we're choosing. Maybe something for the group to decide in a meeting.